plugins/async-node/components/src/dsl/async.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import type { ObjectNode } from "react-json-reconciler";
+
+export type AsyncNodeProps = {
+  id?: string;
+};
+
+export const AsyncNode = React.forwardRef<ObjectNode, AsyncNodeProps>(
+  (props) => {
+    const { id } = props;
+    const Wrapper = React.Fragment;
+
+    return (
+      <Wrapper>
```

Comment:
I would just use `React.Fragment` directly here instead of defining it in a `const`

Reply:
good catch ! done